app/views/admin/budgets/_group_form.html.erb

```diff
@@ -9,20 +9,37 @@
                       placeholder: t("admin.budgets.form.group"),
                       class: "input-group-field" %>
 
-    <% if group.persisted? %>
-      <div class="small-12 medium-6 large-4">
-        <%= f.label :name, t("admin.budgets.form.max_votable_headings") %>
+    <div class="input-group-button">
+      <%= f.submit button_title, class: "button success" %>
+    </div>
+  </div>
 
+  <% if group.persisted? %>
+    <div class="row">
+      <div class="columns small-12 medium-4 large-4">
```

Comment:
Hi @juan-fdz-hawa! the classes on Foundation are hereditary (so large is no needed if it's the same of medium) can you change this to `small-12 medium-4 column` (column and columns works fine, but it's for maintain more consistency with the rest of app) 🙏 😌

Reply:
Gotcha, thanks!